roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh

```diff
@@ -43,9 +43,19 @@ EOF
     fi
     # zero out our upstream servers list and feed it into dnsmasq
     echo '' > /etc/dnsmasq.d/origin-upstream-dns.conf
-    for ns in ${DHCP4_DOMAIN_NAME_SERVERS}; do
-       echo "server=${ns}" >> /etc/dnsmasq.d/origin-upstream-dns.conf
-    done
+
+    if [ -n "${DHCP4_DOMAIN_NAME_SERVERS}" ]; then
+        for ns in ${DHCP4_DOMAIN_NAME_SERVERS}; do
+            echo "server=${ns}" >> /etc/dnsmasq.d/origin-upstream-dns.conf
+        done
+    fi
```

Comment:
Lets just get rid of this block and then I'm fine with taking this PR over mine.